content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,455 @@ categories:
+JSON.SET docs:01 $ '{"vector":[0.34,0.63,-0.54,-0.69,0.98,0.61], "foo": "bar"}'
 ```
 
-As of v2.6.1, JSON supports multi-value indexing. This capability accounts for vectors as well. Thus, it is possible to index multiple vectors under the same JSONPath. Additional information is available in the [Indexing JSON documents]({{< baseurl >}}/develop/interact/search-and-query/indexing/#index-json-arrays-as-vector) section. 
+One of the additional benefits of JSON is schema flexibility. As of v2.6.1, JSON supports multi-value indexing.
+Thus, it is possible to index multiple vectors under the same JSONPath.
+
+Here are some examples of multi-value indexing with vectors:
 
 **Example**
 ```
-JSON.SET 1 $ '{"vec":[[1,2,3,4], [5,6,7,8]]}'
-JSON.SET 1 $ '{"foo":{"vec":[1,2,3,4]}, "bar":{"vec":[5,6,7,8]}}'
+JSON.SET key $ '{"vector":[[1,2,3,4], [5,6,7,8]]}'
+JSON.SET key $ '{"item1":{"vector":[1,2,3,4]}, "item2":{"vector":[5,6,7,8]}}'
 ```
 
-## Querying vector fields
+Additional information and examples are available in the [Indexing JSON documents]({{< baseurl >}}/develop/interact/search-and-query/indexing/#index-json-arrays-as-vector) section.
 
-You can use vector similarity queries in the [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search/) query command. To use a vector similarity query, you must specify the option `DIALECT 2` or greater in the command itself, or set the `DEFAULT_DIALECT` option to `2` or greater, by either using the command [`FT.CONFIG SET`]({{< baseurl >}}/commands/ft.config-set/) or when loading the `redisearch` module and passing it the argument `DEFAULT_DIALECT 2`.
+## Search with vectors
+You can run vector search queries with the [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search/) or [`FT.AGGREGATE`]({{< baseurl >}}/commands/ft.aggregate/) commands.
 
-There are two types of vector queries: *KNN* and *range*:
+To use a vector similarity query with `FT.SEARCH`, you must always specify the `DIALECT 2` option. See the [dialects page]({{< relref "/develop/interact/search-and-query/advanced-concepts/dialects" >}}) for more information.
 
-### KNN search
-The syntax for vector similarity KNN queries is `*=>[<vector_similarity_query>]` for running the query on an entire vector field, or `<primary_filter_query>=>[<vector_similarity_query>]` for running similarity query on the result of the primary filter query. 
+### Standard vector search
 
-As of version 2.4, you can use `vector_similarity_query` only per query and over the entire query filter.
+Standard vector search has the following common syntax:
 
-**Invalid example** 
+**Syntax**
 
-`"(@title:Matrix)=>[KNN 10 @v $B] @year:[2020 2022]"`
+```
+FT.SEARCH <index_name>
+  <primary_filter_query>=>[KNN <top_k> @<vector_field> $<vector_blob_param> $<vector_query_params> AS <distance_field>]
+  PARAMS <vector_query_params_count> [<vector_query_param_name> <vector_query_param_value> ...]
+  SORTBY <distance_field>
+  DIALECT 2
+```
+TODO ^^ Need to validate this query generalization
```

Comment:
@tylerhutcherson `PARAMS` is a general purpose keyword for parameterized queries so `<vector_query_params_{word}>` need to be changed to `<query_params_{word}>` . Also you must send your blob here (not present in the above example)